ood.yaml

```diff
@@ -2,3 +2,4 @@
 - hosts: ood
   roles:
     - ood
+    - { name: 'oodmatlab', tags: 'oodmatlab' }
```

Comment:
Let's use a similar naming convention.  If we are calling something ohpc_matlab above then this would more naturally be ood_matlab.